projects/plugins/boost/tests/e2e/jest.config.js

```diff
@@ -0,0 +1,43 @@
+/**
+ * For a detailed explanation of configuration properties, visit:
+ * https://jestjs.io/docs/en/configuration.html
+ */
+
+if ( process.env.E2E_DEBUG ) {
+	process.env.DEBUG = 'pw:browser|api|error';
+	process.env.PWDEBUG = 1;
+}
+
+module.exports = {
+	testEnvironment: '<rootDir>/node_modules/jetpack-e2e-tests/lib/env/playwright-environment.js',
+	globalSetup: '<rootDir>/node_modules/jetpack-e2e-tests/lib/env/global-setup.js',
+	globalTeardown: '<rootDir>/node_modules/jetpack-e2e-tests/lib/env/global-teardown.js',
+	setupFilesAfterEnv: [
+		'<rootDir>/node_modules/jetpack-e2e-tests/jest.setup.js',
+		'expect-playwright',
+	],
+	testRunner: 'jest-circus/runner',
+	runner: 'groups',
+	testEnvironmentOptions: {
+		resultsDir: 'output/allure-results',
+	},
+	reporters: [
+		'default',
+		[
+			'jest-junit',
+			{
+				suiteName: 'Jetpack Boost E2E tests',
```

Comment:
I wonder if it makes sense to duplicate everything in jest.config. I believe only suite name is different in the reporters. Maybe you can copy the original file in a postinstall task as with default config file?

Reply:
After a few trials and errors, I actually took still another approach. I managed to extend the `jest.config.js` file from Jetpack into this the one in Jetpack Boost and I had to fix issues with the path of the other config files used by Jest. If in the `jest.config.js` file inside Boost i tried something like:

```
const sharedConfig = require( `./${ jetpackTestsRootDir }/jest.config` );

module.exports = {
	rootDir: './node_modules/jetpack-e2e-tests',
	...sharedConfig,
};
```

it still did not work because then it could not find the Jetpack Boost local tests files. I tried to play with the `roots` and `projects` settings too but could not make it work either. So my final solution was to get the shared config to rename the paths (and at the same time the tests suite label). It seems to work well. 

I have also extended the `babel.config.js` as a local Babel config file needs to be in the Jetpack Boost e2e tests directory.

This is being done in commit 25d8cd3f8317fcf04782c88b7ec1f4008ed75bfe

What do you think of this approach?